package.json

```diff
@@ -4,10 +4,11 @@
   "description": "Solidity GMP SDK and utilities provided by Axelar for cross-chain development",
   "main": "index.js",
   "scripts": {
-    "build": "npx hardhat clean && npx hardhat compile && npm run copy:interfaces",
+    "build": "npx hardhat clean && npx hardhat compile && npm run copy:interfaces && npm run clean:artifacts",
```

Comment:
We only wanna clean artifacts during the gh action. Build info is useful for contract verification for e.g. Also, tests are failing due to this.

Reply:
Fixed